test/framework/src/main/java/org/elasticsearch/test/AbstractSerializingTestCase.java

```diff
@@ -60,15 +61,9 @@ public void testFromXContent() throws IOException {
     public void testToString() throws IOException {
         for (int runs = 0; runs < NUMBER_OF_TEST_RUNS; runs++) {
             T testInstance = createTestInstance();
-            if (testInstance instanceof ToXContentToBytes) {
-                XContentType xContentType = XContentType.JSON;
-                String toString = testInstance.toString();
-                assertParsedInstance(xContentType, new BytesArray(toString), testInstance);
-                for (Map.Entry<String, T> alternateVersion : getAlternateVersions().entrySet()) {
-                    String instanceAsString = alternateVersion.getKey();
-                    assertParsedInstance(XContentType.JSON, new BytesArray(instanceAsString), alternateVersion.getValue());
-                }
-            }
+            XContentType xContentType = XContentType.JSON;
+            String toString = Strings.toString(testInstance);
```

Comment:
The difference here from the query and aggs test cases is that we have no guarantee that `toString` is implemented on the class under test. In fact in most cases it probably isn't implemented. I can check if the test class implements `ToXContentToBytes` and call `toString` in that case but I'm not sure there is much else I can do?